packages/components/src/modal/index.js

```diff
@@ -108,8 +101,6 @@ class Modal extends Component {
 	 */
 	render() {
 		const {
-			overlayClassName,
-			className,
 			onRequestClose,
```

Comment:
In the README file we keep those two props documented:
- `overlayClassName`
- `className`
See https://github.com/WordPress/gutenberg/tree/d10fa3862f79e11206569b379909e6bbac124fb8/packages/components/src/modal#props.

Should we keep those two props explicitly listed here for documentation purposes? I guess we use `otherProps` to pass other props as well. Example: `role`. What do you think?

Reply:
Good call, I'll make that more explicit.

Reply:
On closer investigation it looks like there are lots of props that are handled by `ModalFrame`, some like `shouldCloseOnEsc` are only used in the component's methods. I've added a comment to `Modal` to explain that they're passed through to `ModalFrame` rather than explicitly destructuring them.

Reply:
I was afraid that it is the case already. It's fine as is. Fortunately, we don't auto-generate docs for components 😅